share/scripts/inverse/csg_table

```diff
@@ -96,6 +96,10 @@ postadd overwrite postadd_overwrite.sh
 postadd plot postadd_plot.sh
 postadd average postadd_average.sh
 
+# smoothing for updated potential to allow forgetting noise from previous iterations (rdfs)
+# the script is the same as postupd_smooth.sh
+postadd smooth postadd_smooth.sh
```

Comment:
I understand, this doesn't replace the smoothing the dU update, but I am wondering if we need this at all. Plus, it has nothing to do with the `multidir` feature, so let's do this in another pull request (and have the discussion there.)

(Background earlier version of VOTCA has a `postupd smooth`, but users always used it wrong and got a very washed out, feature-less, potentials.)